src/components/tooltip/tooltip.module.scss

```diff
@@ -17,42 +17,56 @@ $triangle-color: var(--rp-ui-base-almost-black-light);
 
 .tooltip {
   transition: opacity 300ms ease;
-  margin: $DEFAULT_MARGIN;
+  z-index: 9;
```

Comment:
Could you please describe why did you select this exact value?

Reply:
Done.